src/controller/proxy/local.go

```diff
@@ -46,15 +49,21 @@ type localInterface interface {
 	CheckDependencies(ctx context.Context, repo string, man distribution.Manifest) []distribution.Descriptor
 	// DeleteManifest cleanup delete tag from local cache
 	DeleteManifest(repo, ref string)
+	// SameArtifact check if same artifact in local
+	SameArtifact(ctx context.Context, repo, tag, dig string) (bool, error)
```

Comment:
This func and `ManifestExists` can be merged into one?

Reply:
Removed